scripts/travis.sh

```diff
@@ -33,14 +33,22 @@ function fold_end() {
 source ./scripts/retry.sh
 retry 3 yarn install
 
+# For socketserver
+pip3 install websockets
```

Comment:
This is yet another requirement, that pip3 is installed on dev machine.
No issue for me, but something I am wary of.